agents/python/requirements.in

```diff
@@ -0,0 +1,59 @@
+opentelemetry-distro==0.45b0
+# We don't use the distro[otlp] option which automatically includes exporters since gRPC is not appropriate for
+# injected auto-instrumentation, where it has a strict dependency on the OS / Python version the artifact is built for.
+opentelemetry-exporter-otlp-proto-http==1.24.0
+
+opentelemetry-propagator-b3==1.24.0
+opentelemetry-propagator-jaeger==1.24.0
+opentelemetry-propagator-aws-xray==1.0.1
```

Comment:
are these used in practice?
in node.js we don't have them. wonder if we should add them by default or not